.github/workflows/coding.yml

```
@@ -55,4 +55,30 @@
+      - name: Run tests - TypeScript Challenges
+        run: |
+          cd programming/typescript/challenges
+          npm install
+          npm run build
+          npm test
+      # - name: Upload coverage to Codecov
+      #   uses: codecov/codecov-action@v4
+      #   env:
+      #     CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+      #   with:
+      #     fail_ci_if_error: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)